cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,156 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+
+.. :section-numbering: true
+
+.. _admin-installation-azure-hdinsight:
+
+=========================================
+Installation on Microsoft Azure HDInsight
+=========================================
+
+Introduction
+============
+
+This section describes installing CDAP on `Microsoft Azure HDInsight
+<https://azure.microsoft.com/en-us/services/hdinsight/>`__ using
+Microsoft's website to:
+
+- Create an HBase cluster;
+- Install CDAP as an application on the cluster; and then
+- Verify that CDAP is installed and running by using the CDAP UI.
+
+Information on Microsoft Azure HDInsight is `available online
+<https://docs.microsoft.com/en-us/azure/hdinsight/>`__.
+
+CDAP |short-version| is compatible with Microsoft Azure HDInsight.
+
+
+Creating the Cluster
+====================
+
+1. Log onto Microsoft Azure and navigate to *HDInsight clusters* (under *Intelligence + analytics*):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-1.png
+      :figwidth: 100%
+      :width: 611px
+      :class: bordered-image
+
+      **Microsoft Azure:** Finding HDInsight
+
+#. Create an *HDInsight cluster* by clicking "Create". Enter a *Cluster name* (such as
+   *cdap-cluster*). Under *Cluster type*, select "HBase". (To continue, click *Select* to
+   close the *Cluster configuration* window):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster
+
+#. Enter the required basic settings: *Cluster login password*, *Resource group*, and *Location*:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.png
+      :figwidth: 100%
+      :width: 674px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Configuring basic settings
+
+#. Under *Storage*, set a storage account, either an existing account or enter a unique name for a new account.
+
+#. Click *Next* to review the summary.  Then click *Create* to start cluster initialization:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-4.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Summary and confirming configuration
+
+#. Once the cluster is ready, its icon will show on the portal dashboard:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-5.png
+      :figwidth: 100%
+      :width: 176px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Cluster is created and running
+
+
+Install CDAP as an Application
```

Comment:
This doesn't have to be a second step.  You can add CDAP as an application along with the initial cluster provisioning.  This is a big change to this doc, but I think its worth it